compiler/config.go

```diff
@@ -0,0 +1,70 @@
+package compiler
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"encr.dev/compiler/internal/cueutil"
+	"encr.dev/parser/est"
+	"encr.dev/pkg/eerror"
+	"encr.dev/pkg/vfs"
+)
+
+// pickupConfigFiles pulls any user configuration files off the filesystem
+// and into a virtual file system, which then can be used to generate the runtime
+// configuration for each service
+func (b *builder) pickupConfigFiles() error {
+	// Create a virtual filesystem for the config files
+	configFiles, err := vfs.FromDir(b.appRoot, func(path string) bool {
+		// any CUE files
+		if filepath.Ext(path) == ".cue" {
+			return true
+		}
+
+		// Pickup any files within a CUE module folder (either at the root of the app or in a subfolder)
+		if strings.Contains(path, "/cue.mod/") || strings.HasPrefix(path, "cue.mod/") {
```

Comment:
The ability to import config from other packages (like a shared library package).

Even though Encore itself doesn't use `cue.mod` yet, users _might_ and we need to pick up the files otherwise stuff which works locally might not work when deployed to the cloud.